src/components/Recommendations.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import recommendations from './definitions/recommendationsCopy';
+
+const Recommendations = () => {
+  return (
+    <div>
+      {recommendations.map((recommendation) => {
+        const recommendationSplits = recommendation.text.split('\n');
+
+        return (
+          <div key={recommendation.name}>
+            <a target="_blank" href={recommendation.linkedInUrl}>
+              <img src={recommendation.profilePictureUrl} className="picture" />
+            </a>
+            <h1>{recommendation.name}</h1>
```

Comment:
Generally, an `h1` should be reserved for the site or page title, so something like an `h2` might work better here.

Alternatively, since there are multiple recommendations, I would just do on `h2/h3` for the `name` and set the others as `p` with different styling to convert the level of importance visually. Having a lot of `h2/h3` element son the page isn't great in terms of conveying the level of importance of different elements for users with accessibility needs.